src/routes/notification-page/index.jsx

```diff
@@ -0,0 +1,11 @@
+import Notifications from '../../components/notifications/notifications';
+
+const NotificationPage = () => {
```

Comment:
`NotificationStory` make sense.

```suggestion
const NotificationStory = () => {
```